src/test/java/bowling/domain/NormalFrameTest.java

```diff
@@ -1,32 +1,84 @@
 package bowling.domain;
 
-import static bowling.domain.Pins.START_PIN_COUNT;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
+import bowling.domain.state.FirstBowl;
+import bowling.domain.state.Miss;
+import bowling.domain.state.Spare;
+import bowling.domain.state.Strike;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-class FrameTest {
+class NormalFrameTest {
 
     private Frame frame;
 
     @BeforeEach
     void setUp() {
-        frame = new Frame();
+        frame = new NormalFrame(null, null);
     }
 
+    @Test
+    void 프레임의_최초_준비상태에서_한번던진상태로_간다() {
+        frame.shot(1);
+        assertThat(frame.state()).isOfAnyClassIn(FirstBowl.class);
```

Comment:
`isInstanceOf()`로 특정 클래스 타입인지를 명확하게 확인해보는 건 어떨까요?
`isOfAnyClassIn()`은 전달된 클래스 타입 중 하나라도 일치하면 통과하기 때문에, 
만약 실수로 여러 클래스가 전달되는 경우가 생기면 테스트의 결과가 명확해지지 않다고 생각해요.
```suggestion
        assertThat(frame.state()).isInstanceOf(FirstBowl.class);
```

-----
  
다음 코드를 실행해도 테스트가 통과합니다만, 좋은 테스트라고 보긴 어려워보입니다.
```
assertThat(frame.state()).isOfAnyClassIn(FirstBowl.class, Miss.class);
```